10up/headstartwp
# feat/570: Enhancements to menu item classes

### Description of the Change



Closes #570

Adds the following:

- Support for custom menu item HTML classes.
- `menu-item-has-children` HTML class for menu item with children.

### How to test the Change


1. Perform the basic setup.
2. Add children to a menu item.
3. Add a custom CSS class to a menu item.
4. Verify on the frontend.

### Changelog Entry

> Added - Support for custom menu item CSS classes.
> Added - `menu-item-has-children` CSS class for menu item with children.

### Credits

Props @burhandodhy @Sidsector9 


### Checklist:
